install/nextcloud/install.yml

```diff
@@ -0,0 +1,15 @@
+id: nextcloud
+name: Nextcloud
+title: Nextcloud
+description: |
+  Integrate your Nextcloud processes with our Infrastructure agent and Prometheus open metric integration to collect data and send it to New Relic.
```

Comment:
"Use our infrastructure agent and the Prometheus open metric integration to monitor the performance of the processes on your Nextcloud server."